src/pages/app-development/starter-kit/customers.md

```diff
@@ -10,4 +10,39 @@ keywords:
  - Tools
 ---
 
-# Customers data flow
+# `customer` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
```

Comment:
Should we include a diagram similar like this one in each one of the data flows? https://wiki.corp.adobe.com/display/EntComm/Product+Data+Object